docs/100-upgrade/migration-guides/3.2-3.3.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 3.2 -> 3.3
+sidebar_position: 98 # decrease for each release
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.2
+  to 3.3.
+---
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/\*@3.3.0"
+```
+
+## Code changes
```

Comment:
**issue:** this version requires once again changes to the `entry.server.ts`. This patch doesn't mention them.

**suggestion:** add a section about the `entry.server.ts` update

Reply:
Still not correct sorry. What I meant is that the change in `entry.server.ts` isn't directly related to "Automatic session cookie headers".

As a consequence, it must be in a dedicated section … like for the 3.1 -> 3.2 migration guide.

Reply:
Sorry I misinterpreted "Section". I made a change again, is this what you meant?

Reply:
Yes, perfect. Thanks!

Now, it's a matter of who will merge first between this MR and @paul-vd's one (#832) to prevent the conflict :rocket:

Reply:
I did :D